aws/data_source_aws_ebs_volume.go

```diff
@@ -117,9 +122,19 @@ func mostRecentVolume(volumes []*ec2.Volume) *ec2.Volume {
 	return sortedVolumes[len(sortedVolumes)-1]
 }
 
-func volumeDescriptionAttributes(d *schema.ResourceData, volume *ec2.Volume) error {
+func volumeDescriptionAttributes(d *schema.ResourceData, meta interface{}, volume *ec2.Volume) error {
```

Comment:
I don't think we need the whole `meta` here 🤔 , do you mind "reducing it" to `client *AWSClient`?